kiwi/system/kernel.py

```diff
@@ -21,6 +21,8 @@
 # project
 from ..command import Command
 
+from ..exceptions import KiwiKernelLookupError
```

Comment:
yes, what's interesting here is that I had that in the early days of this code when another python Guru told me it should be better changed to relative imports. Apart from personal taste is there also a technical reason why relative imports sucks ? Thanks